AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6615 @@
+{
```

Comment:
<p>I would group arviz and pymc imports separately.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>You mean importing them only when we need them in the NB?</p>

Reply:
Yes, in this NB I would put the imports where they are introduced, not all at once so theyre "imported" into the students brain when needed